src/main/java/org/italiangrid/storm/webdav/server/servlet/resource/StormDirectoryResourceWrapper.java

```diff
@@ -133,6 +137,8 @@ public String getListHTML(String base) {
               .withIsDirectory(r.isDirectory())
               .withLastModificationTime(Date.from(r.lastModified()))
               .withSizeInBytes(r.length())
+              .withLocality(
+                  pathResolver.getLocality(URIUtil.addEncodedPaths(encodedBase, r.getFileName())))
```

Comment:
In storm-tape we never return locality LOST, not to scare the client. We set it to UNAVAILABLE and log something for the operations to check the status of the file. See https://baltig.infn.it/cnafsd/storm-tape/-/blob/main/src/tape_service.cpp#L76